src/App.tsx

```diff
@@ -4,18 +4,48 @@ import Login from "./pages/Login";
 import SignUp from "./pages/Signup";
 import "bootstrap/dist/css/bootstrap.min.css";
 import Polling from "./pages/Polling";
+import Protected from "./Protected";
+import AddPoll from "./pages/AddPoll";
+import CreateUser from "./pages/CreateUser";
+import ListUser from "./pages/ListUser";
+import ViewPoll from "./pages/ViewPoll";
```

Comment:
All pages should be end with Page
like HomePage, DashboardPage @Rishab-innovative